packages/dds/tree/src/events/events.ts

```diff
@@ -97,26 +100,24 @@ export interface IEventEmitter<E extends Events<E>> {
  * Or, compose over it:
  * @example
  * ```ts
- * class MyClass extends IEventEmitter<MyEvents> {
+ * class MyClass implements IEventEmitter<MyEvents> {
  *   private readonly events = EventEmitter.create<MyEvents>();
  *
  *   private load() {
  *     this.events.emit("loaded");
  *   }
  *
- *   public on<K extends (string | symbol) & keyof MyEvents>(
+ *   public on<K extends keyof MyEvents>(
  *     eventName: K,
  *     listener: MyEvents[K],
  *   ): () => void {
```

Comment:
The extra line wrapping is unneeded and makes the example harder to copy paste out of the comment. I think our formatter prefers the shorter version now anyway:
```suggestion
 *   public on<K extends keyof MyEvents>(eventName: K, listener: MyEvents[K]): () => void {
```